nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-1-0-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/ConsumerLease.java

```diff
@@ -174,7 +177,7 @@ void poll() {
          * This behavior has been fixed via Kafka KIP-62 and available from Kafka client 0.10.1.0.
          */
         try {
-            final ConsumerRecords<byte[], byte[]> records = kafkaConsumer.poll(10);
+            final ConsumerRecords<byte[], byte[]> records = kafkaConsumer.poll(maxWaitMillis / 10);
```

Comment:
Why are we doing ``maxWaitMillis / 10`` instead of just ``maxWaitMillis``?
(sorry for the possibly dumb question here)